pacstall

```diff
@@ -27,7 +27,7 @@ export LC_ALL=C
 export HOMEDIR="/home/$(logname)"
```

Comment:
I think we should remove `HOMEDIR` too, since pacstall is no longer running under root, so people can directly use `~` to access user's home directory.

Reply:
true